schema/v2.yaml

```diff
@@ -102,10 +102,25 @@ models:
       optional: false
       nullable: false
     - field: messageId
-      description: The message id of the quoted message
+      description: |
+        The message id of the quoted message
+
+        If this field is provided, the quote text is being ignored. The quote
+        text should still be filled though, for backwards compatibility.
       type: String
       optional: true
       nullable: false
+    - field: message
+      description: |
+        The quoted message
+
+        This should always be provided if a message ID is specified, unless the
+        app cannot find the message in its database.
```

Comment:
Suggest to add:

> This MUST NOT be a message containing a quote!

Reply:
As long as quotes aren't resolved recursively, there should be little harm in doing so. I think it's better if Threema Web is written in a robust way that can handle this, versus having to trust that the app doesn't have any validation bug.

Reply:
I disagree. There's no use case for a quote within a quote.

Reply:
I also think the recursive resolving is a very easy mistake to make.